### PR TITLE
Programs Event Framework

### PR DESCRIPTION
**Pull request by ygerasimov**
recreating PR from ygerasimov username.

### CODE REVIEW

File: modules/custom/openy_gxp/openy_gxp_example/openy_gxp_example.module [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+    $locations = array_values($query->execute());
+  }
+
+  $row['locations'] = $locations[rand(0, count($locations) - 1)];
+}
+
+///**
+// * Implements hook_openy_gxp_categories_csv_row_alter().
+// */
+//function openy_gxp_example_openy_gxp_categories_csv_row_alter(&$row) {
```
Comment (podarok) [inline modules/custom/openy_gxp/openy_gxp_example/openy_gxp_example.module]: remove commented code, please

File: modules/custom/openy_gxp/src/Plugin/migrate/process/OpenYGXPCategory.php [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+ * )
+ */
+class OpenYGXPCategory extends ProcessPluginBase {
+
+  /**
+   * {@inheritdoc}
+   */
+  public function transform($value, MigrateExecutableInterface $migrate_executable, Row $row, $destination_property) {
+    $value = json_decode($value, TRUE);
+//    {"title":"Cardio","description":"A great cardio experience...","activity":119}
```
Comment (podarok) [inline modules/custom/openy_gxp/src/Plugin/migrate/process/OpenYGXPCategory.php]: remove commented code, please

File: modules/custom/openy_repeat/js/repeat.js [hunk 3]
```diff
@@ -0,0 +1,230 @@
...
+      //Results
+    },
+    mounted() {
+      runAjaxRequest(this, currentDate, eventLocation, eventCategory);
+      changeDateTitle(currentDate);
+      updateAtc();
+    },
+    watch: {
+      'globalData.date': function(newValue, oldValue) {
+        // this.$root.mounted();
```
Comment (podarok) [inline modules/custom/openy_repeat/js/repeat.js]: remove commented code, please

File: modules/custom/openy_repeat/js/vue/vue-router.js [hunk 4]
```diff
@@ -0,0 +1,2631 @@
+/**
+  * vue-router v3.0.1
```
Comment (podarok) [inline modules/custom/openy_repeat/js/vue/vue-router.js:2]: is it possible to add vue via composer into vendors  folder?

File: modules/custom/openy_repeat/openy_repeat.libraries.yml [hunk 5]
```diff
@@ -0,0 +1,51 @@
+openy_repeat:
+  version: VERSION
```
Comment (podarok) [inline modules/custom/openy_repeat/openy_repeat.libraries.yml]: provide versions everywhere, please
Use 0.1 as initial

File: modules/custom/openy_repeat/openy_repeat.libraries.yml [hunk 6]
```diff
@@ -0,0 +1,51 @@
...
+      css/modal.css: {}
+  dependencies:
+    - openy_repeat/openy_repeat.vue
+    - openy_repeat/openy_repeat.vue_router
+    - openy_repeat/openy_repeat.vue.table
+    - openy_repeat/open_repeat.moment
+    - openy_repeat/open_repeat.bootstrap-datepicker
+
+openy_repeat.vue:
+  version: VERSION
```
Comment (podarok) [inline modules/custom/openy_repeat/openy_repeat.libraries.yml]: provide versions everywhere, please
Use 0.1 as initial

File: modules/custom/openy_repeat/openy_repeat.libraries.yml [hunk 7]
```diff
@@ -0,0 +1,51 @@
...
+  js:
+    js/vue/vue.min.js: {}
+
+openy_repeat.vue_router:
+  version: VERSION
```
Comment (podarok) [inline modules/custom/openy_repeat/openy_repeat.libraries.yml]: provide versions everywhere, please
Use 0.1 as initial

File: modules/custom/openy_repeat/openy_repeat.libraries.yml [hunk 8]
```diff
@@ -0,0 +1,51 @@
...
+  js:
+    js/vue/vue-router.js: {}
+
+openy_repeat.vue.table:
+  version: VERSION
```
Comment (podarok) [inline modules/custom/openy_repeat/openy_repeat.libraries.yml]: provide versions everywhere, please
Use 0.1 as initial

File: modules/custom/openy_repeat/openy_repeat.libraries.yml [hunk 9]
```diff
@@ -0,0 +1,51 @@
...
+  js:
+    js/components/event_table.js: {}
+
+open_repeat.moment:
+  version: VERSION
```
Comment (podarok) [inline modules/custom/openy_repeat/openy_repeat.libraries.yml]: provide versions everywhere, please
Use 0.1 as initial

File: modules/custom/openy_repeat/openy_repeat.libraries.yml [hunk 10]
```diff
@@ -0,0 +1,51 @@
...
+  js:
+    https://cdnjs.cloudflare.com/ajax/libs/moment.js/2.13.0/moment.min.js: { type: external }
+
+open_repeat.bootstrap-datepicker:
+  version: VERSION
```
Comment (podarok) [inline modules/custom/openy_repeat/openy_repeat.libraries.yml]: provide versions everywhere, please
Use 0.1 as initial

File: modules/custom/openy_repeat/openy_repeat.libraries.yml [hunk 11]
```diff
@@ -0,0 +1,51 @@
...
+  js:
+    https://cdnjs.cloudflare.com/ajax/libs/bootstrap-datepicker/1.3.0/js/bootstrap-datepicker.js: { type: external }
+  css:
+    base:
+      'https://cdnjs.cloudflare.com/ajax/libs/bootstrap-datepicker/1.3.0/css/datepicker.css': { type: external }
+
+openy_repeat.locations:
+  version: VERSION
```
Comment (podarok) [inline modules/custom/openy_repeat/openy_repeat.libraries.yml]: provide versions everywhere, please
Use 0.1 as initial

File: modules/custom/openy_repeat/src/Controller/RepeatController.php [hunk 12]
```diff
@@ -0,0 +1,213 @@
...
+   * @return array
+   */
+  public function getLocations() {
+    $sql = "SELECT DISTINCT nd.title as location
+            FROM {node} n
+            INNER JOIN node__field_session_location l ON n.nid = l.entity_id AND l.bundle = 'session'
+            INNER JOIN node_field_data nd ON l.field_session_location_target_id = nd.nid
+            WHERE n.type = 'session'";
+
+    $connection = \Drupal::database();
```
Comment (podarok) [inline modules/custom/openy_repeat/src/Controller/RepeatController.php:129]: pass database via DI, please

File: modules/custom/openy_repeat/src/Controller/RepeatController.php [hunk 13]
```diff
@@ -0,0 +1,214 @@
...
+  }
+
+  /**
+   * Get detailed info about Class.
+   */
+  public function getClassesInfo() {
+    $nids = \Drupal::entityQuery('node')
+      ->condition('type','class')
+      ->execute();
+    $classes = Node::loadMultiple($nids);
```
Comment (podarok) [inline modules/custom/openy_repeat/src/Controller/RepeatController.php:173]: we can have tons of classes
loadmultiple is not an option, especially when you foreach array 
please splice array by 50 maximum

File: modules/custom/openy_repeat/src/Plugin/Block/RepeatSchedulesBlock.php [hunk 14]
```diff
@@ -0,0 +1,78 @@
...
+   * @return array
+   */
+  public function getLocations() {
+    $sql = "SELECT DISTINCT nd.title as location 
+            FROM {node} n
+            INNER JOIN node__field_session_location l ON n.nid = l.entity_id AND l.bundle = 'session'
+            INNER JOIN node_field_data nd ON l.field_session_location_target_id = nd.nid
+            WHERE n.type = 'session'";
+
+    $connection = \Drupal::database();
```
Comment (podarok) [inline modules/custom/openy_repeat/src/Plugin/Block/RepeatSchedulesBlock.php:30]: pass via DI, please
Take care about performance

File: modules/custom/openy_repeat/src/Plugin/Block/RepeatSchedulesBlock.php [hunk 15]
```diff
@@ -0,0 +1,78 @@
...
+   * @return array
+   */
+  public function getCategories() {
+    $sql = "SELECT title 
+            FROM {node_field_data} n
+            WHERE n.type = 'activity'
+            AND n.status = '1'
+            ORDER BY title ASC";
+
+    $connection = \Drupal::database();
```
Comment (podarok) [inline modules/custom/openy_repeat/src/Plugin/Block/RepeatSchedulesBlock.php:48]: DI

File: modules/custom/openy_repeat/src/Plugin/Block/RepeatSchedulesBlock.php [hunk 16]
```diff
@@ -0,0 +1,78 @@
...
+    $query = $connection->query($sql);
+    return $query->fetchCol();
+  }
+
+  /**
+   * {@inheritdoc}
+   */
+  public function build() {
+    $request_stack = \Drupal::service('request_stack');
```
Comment (podarok) [inline modules/custom/openy_repeat/src/Plugin/Block/RepeatSchedulesBlock.php:57]: DI, please

File: modules/custom/openy_repeat/src/Plugin/Block/RepeatSchedulesLocBlock.php [hunk 17]
```diff
@@ -0,0 +1,37 @@
...
+  /**
+   * {@inheritdoc}
+   */
+  public function build() {
+    $sql = "SELECT DISTINCT nd.title as location 
+            FROM {node} n
+            INNER JOIN node__field_session_location l ON n.nid = l.entity_id AND l.bundle = 'session'
+            INNER JOIN node_field_data nd ON l.field_session_location_target_id = nd.nid
+            WHERE n.type = 'session'";
+    $connection = \Drupal::database();
```
Comment (podarok) [inline modules/custom/openy_repeat/src/Plugin/Block/RepeatSchedulesLocBlock.php:27]: DI, please

File: modules/custom/openy_repeat/src/RepeatManager.php [hunk 18]
```diff
@@ -0,0 +1,640 @@
...
+    $this->storage->delete($session_instances);
+
+    return count($session_instances);
+  }
+
+  /**
+   * {@inheritdoc}
+   */
+  public function getSessionData(NodeInterface $session) {
+    $moderation_wrapper = Drupal::service('openy_moderation_wrapper.entity_moderation_status');
```
Comment (podarok) [inline modules/custom/openy_repeat/src/RepeatManager.php:123]: DI, please

File: modules/custom/openy_repeat_entity/src/Entity/RepeatInterface.php [hunk 19]
```diff
@@ -0,0 +1,13 @@
...
+
+use Drupal\Core\Entity\ContentEntityInterface;
+
+/**
+ * Provides an interface defining a Repeat entity.
+ * @ingroup openy_repeat_entity
+ */
+interface RepeatInterface extends ContentEntityInterface {
+
+}
```
Comment (podarok) [inline modules/custom/openy_repeat_entity/src/Entity/RepeatInterface.php:13]: please make all files 644, not 755
modules/custom/openy_schedules/js/openy_schedules.js

File: modules/openy_features/openy_demo_content/modules/openy_demo_nlanding/openy_demo_nlanding.module [hunk 20]
```diff
@@ -0,0 +1,125 @@
...
+      'field_prgf_repeat_lschedules_ref' => ['plugin_id' => 'repeat_schedules_loc_block'],
+      'field_prgf_repeat_lschedules_prf' => ['target_id' => $schedules_node->id()]
+    ]
+  );
+  $location_paragraph->save();
+
+  $locations_node->field_content->entity = $location_paragraph;
+  $locations_node->save();
+
+  $parent_menu_link = \Drupal\menu_link_content\Entity\MenuLinkContent::load(39);
```
Comment (podarok) [inline modules/openy_features/openy_demo_content/modules/openy_demo_nlanding/openy_demo_nlanding.module:82]: remove hardcoded ID

File: modules/openy_features/openy_node/modules/openy_node_session/config/install/core.entity_form_display.paragraph.session_time.default.yml [hunk 21]
```diff
@@ -24,6 +25,12 @@ content:
     third_party_settings: {  }
     type: options_buttons
     region: content
+  moderation_state:
```
Comment (podarok) [inline modules/openy_features/openy_node/modules/openy_node_session/config/install/core.entity_form_display.paragraph.session_time.default.yml:28]: remove moderation dependency. We shouldn't moderate paragraphs globally

File: modules/openy_features/openy_node/modules/openy_node_session/config/install/field.field.node.session.field_session_instructor.yml [hunk 22]
```diff
@@ -0,0 +1,24 @@
+langcode: en
+status: true
+dependencies:
+  config:
+    - field.storage.node.field_session_instructor
+    - node.type.session
+  module:
+    - datalayer
```
Comment (podarok) [inline modules/openy_features/openy_node/modules/openy_node_session/config/install/field.field.node.session.field_session_instructor.yml:8]: datalayer is not used, no need for dependency

File: modules/openy_features/openy_node/modules/openy_node_session/config/install/field.field.node.session.field_session_plocation.yml [hunk 23]
```diff
@@ -5,11 +5,17 @@ dependencies:
     - field.storage.node.field_session_plocation
     - node.type.facility
     - node.type.session
+  module:
+    - datalayer
+third_party_settings:
+  datalayer:
+    expose: 0
```
Comment (podarok) [inline modules/openy_features/openy_node/modules/openy_node_session/config/install/field.field.node.session.field_session_plocation.yml:12]: datalayer is not used, no need for dependency

File: modules/openy_features/openy_node/modules/openy_node_session/openy_node_session.info.yml [hunk 24]
```diff
@@ -4,6 +4,8 @@ type: module
 core: 8.x
 package: OpenY
 dependencies:
+  - content_moderation
+  - datalayer
```
Comment (podarok) [inline modules/openy_features/openy_node/modules/openy_node_session/openy_node_session.info.yml:8]: both not needed
add module version, please

File: modules/openy_features/openy_prgf/modules/openy_prgf_repeat_schedules/config/install/field.field.paragraph.repeat_schedules.field_prgf_repeat_schedule_categ.yml [hunk 25]
```diff
@@ -0,0 +1,33 @@
...
+dependencies:
+  config:
+    - field.storage.paragraph.field_prgf_repeat_schedule_categ
+    - node.type.activity
+    - paragraphs.paragraphs_type.repeat_schedules
+  module:
+    - datalayer
+third_party_settings:
+  datalayer:
+    expose: 0
```
Comment (podarok) [inline modules/openy_features/openy_prgf/modules/openy_prgf_repeat_schedules/config/install/field.field.paragraph.repeat_schedules.field_prgf_repeat_schedule_categ.yml]: not used datalayer, no need for dependency

File: modules/openy_features/openy_prgf/modules/openy_prgf_repeat_schedules/config/install/field.field.paragraph.repeat_schedules.field_prgf_repeat_schedule_excl.yml [hunk 26]
```diff
@@ -0,0 +1,33 @@
...
+dependencies:
+  config:
+    - field.storage.paragraph.field_prgf_repeat_schedule_excl
+    - node.type.activity
+    - paragraphs.paragraphs_type.repeat_schedules
+  module:
+    - datalayer
+third_party_settings:
+  datalayer:
+    expose: 0
```
Comment (podarok) [inline modules/openy_features/openy_prgf/modules/openy_prgf_repeat_schedules/config/install/field.field.paragraph.repeat_schedules.field_prgf_repeat_schedule_excl.yml]: remove datalayer dependency, please

File: modules/openy_features/openy_prgf/modules/openy_prgf_repeat_schedules/config/install/field.field.paragraph.repeat_schedules.field_prgf_repeat_schedule_instr.yml [hunk 27]
```diff
@@ -0,0 +1,28 @@
...
+status: true
+dependencies:
+  config:
+    - field.storage.paragraph.field_prgf_repeat_schedule_instr
+    - paragraphs.paragraphs_type.repeat_schedules
+  module:
+    - datalayer
+third_party_settings:
+  datalayer:
+    expose: 0
```
Comment (podarok) [inline modules/openy_features/openy_prgf/modules/openy_prgf_repeat_schedules/config/install/field.field.paragraph.repeat_schedules.field_prgf_repeat_schedule_instr.yml]: remove datalayer dependency globally, please

File: modules/openy_features/openy_prgf/modules/openy_prgf_repeat_schedules/config/install/paragraphs.paragraphs_type.repeat_schedules.yml [hunk 28]
```diff
@@ -0,0 +1,8 @@
+langcode: en
+status: true
+dependencies: {  }
+id: repeat_schedules
+label: 'Repeat Schedules'
+icon_uuid: null
+description: null
```
Comment (podarok) [inline modules/openy_features/openy_prgf/modules/openy_prgf_repeat_schedules/config/install/paragraphs.paragraphs_type.repeat_schedules.yml]: add descriptions for all paragraps

File: modules/openy_features/openy_prgf/modules/openy_prgf_repeat_schedules/openy_prgf_repeat_schedules.info.yml [hunk 29]
```diff
@@ -0,0 +1,17 @@
+name: 'OpenY Paragraph Repeat Schedules'
+description: 'Repeat Schedules paragraph.'
+type: module
+core: 8.x
+package: OpenY
+dependencies:
+  - datalayer
```
Comment (podarok) [inline modules/openy_features/openy_prgf/modules/openy_prgf_repeat_schedules/openy_prgf_repeat_schedules.info.yml]: not used, remove please

File: modules/openy_features/openy_prgf/modules/openy_prgf_repeat_schedules_loc/config/install/field.field.paragraph.repeat_schedules.field_prgf_repeat_lschedules_prf.yml [hunk 30]
```diff
@@ -0,0 +1,33 @@
...
+dependencies:
+  config:
+    - field.storage.paragraph.field_prgf_repeat_lschedules_prf
+    - node.type.landing_page
+    - paragraphs.paragraphs_type.repeat_schedules_loc
+  module:
+    - datalayer
+third_party_settings:
+  datalayer:
+    expose: 0
```
Comment (podarok) [inline modules/openy_features/openy_prgf/modules/openy_prgf_repeat_schedules_loc/config/install/field.field.paragraph.repeat_schedules.field_prgf_repeat_lschedules_prf.yml]: not used, remove dependency, please

File: modules/openy_features/openy_prgf/modules/openy_prgf_repeat_schedules_loc/config/install/paragraphs.paragraphs_type.repeat_schedules_loc.yml [hunk 31]
```diff
@@ -0,0 +1,8 @@
+langcode: en
+status: true
+dependencies: {  }
+id: repeat_schedules_loc
+label: 'Repeat Schedules Locations'
+icon_uuid: null
+description: null
```
Comment (podarok) [inline modules/openy_features/openy_prgf/modules/openy_prgf_repeat_schedules_loc/config/install/paragraphs.paragraphs_type.repeat_schedules_loc.yml]: provide description, please

File: modules/openy_features/openy_prgf/modules/openy_prgf_repeat_schedules_loc/openy_prgf_repeat_schedules_loc.info.yml [hunk 32]
```diff
@@ -0,0 +1,14 @@
+name: 'OpenY Paragraph Repeat Schedules Locations'
+description: 'Repeat Schedules Locations paragraph.'
+type: module
+core: 8.x
+package: OpenY
+dependencies:
+  - datalayer
```
Comment (podarok) [inline modules/openy_features/openy_prgf/modules/openy_prgf_repeat_schedules_loc/openy_prgf_repeat_schedules_loc.info.yml]: not used, remove, please

File: themes/openy_themes/openy_lily/sass/modules/_schedules.scss [hunk 33]
```diff
@@ -820,9 +815,17 @@ iframe#ga_iframe {
   visibility: hidden;
   position: absolute;
   top: 100%;
-  right: 0;
+  left: 0;
   width: 170px;
   z-index: 900;
+  border: 1px solid #bababa;
```
Comment (podarok) [inline themes/openy_themes/openy_lily/sass/modules/_schedules.scss:821]: colors as variables, please

**Review by podarok — CHANGES_REQUESTED** [review]
Needs some love
Overall - good progress

**Comment by gundevel** [comment]
Build comment file: 
--------------

 
| Environment | Link |
| ------------- | ------------- |
| Fresh OpenY installation - Rose theme | http://openy.cibox.tools/build743 |
| Fresh OpenY installation - Lily theme | http://lily.openy.cibox.tools/build743 |
| Upgraded(upgrade path) installation | http://upgrade.openy.cibox.tools/build743 |
| Installation process | http://install.openy.cibox.tools/build743/install.php |

--------------
Test PASSed.
Refer to this link for build results (access rights to CI server needed): 
http://openy.cibox.tools:8080/job/PR_BUILDER_COMPOSER/743/

**Comment by gundevel** [comment]
Acessibility Sniffer: front page checking WCAG2AA http://openy.cibox.tools/build743/frontWCAG2AAhtmlcs.txt
Acessibility Sniffer: join page checking WCAG2AA http://openy.cibox.tools/build743/joinWCAG2AAhtmlcs.txt
Acessibility Sniffer: locations page checking WCAG2AA http://openy.cibox.tools/build743/locationsWCAG2AAhtmlcs.txt
Acessibility Sniffer: schedules page checking WCAG2AA http://openy.cibox.tools/build743/schedulesWCAG2AAhtmlcs.txt
Acessibility Sniffer: blog page checking WCAG2AA http://openy.cibox.tools/build743/blogWCAG2AAhtmlcs.txt

**Comment by gundevel** [comment]
Build comment file: 
--------------

 
| Environment | Link |
| ------------- | ------------- |
| Fresh OpenY installation - Rose theme | http://openy.cibox.tools/build744 |
| Fresh OpenY installation - Lily theme | http://lily.openy.cibox.tools/build744 |
| Upgraded(upgrade path) installation | http://upgrade.openy.cibox.tools/build744 |
| Installation process | http://install.openy.cibox.tools/build744/install.php |

--------------
Test PASSed.
Refer to this link for build results (access rights to CI server needed): 
http://openy.cibox.tools:8080/job/PR_BUILDER_COMPOSER/744/

**Comment by gundevel** [comment]
Acessibility Sniffer: front page checking WCAG2AA http://openy.cibox.tools/build744/frontWCAG2AAhtmlcs.txt
Acessibility Sniffer: join page checking WCAG2AA http://openy.cibox.tools/build744/joinWCAG2AAhtmlcs.txt
Acessibility Sniffer: locations page checking WCAG2AA http://openy.cibox.tools/build744/locationsWCAG2AAhtmlcs.txt
Acessibility Sniffer: schedules page checking WCAG2AA http://openy.cibox.tools/build744/schedulesWCAG2AAhtmlcs.txt
Acessibility Sniffer: blog page checking WCAG2AA http://openy.cibox.tools/build744/blogWCAG2AAhtmlcs.txt

**Comment by gundevel** [comment]
Build comment file: 
--------------

 
| Environment | Link |
| ------------- | ------------- |
| Fresh OpenY installation - Rose theme | http://openy.cibox.tools/build745 |
| Fresh OpenY installation - Lily theme | http://lily.openy.cibox.tools/build745 |
| Upgraded(upgrade path) installation | http://upgrade.openy.cibox.tools/build745 |
| Installation process | http://install.openy.cibox.tools/build745/install.php |

--------------
Test PASSed.
Refer to this link for build results (access rights to CI server needed): 
http://openy.cibox.tools:8080/job/PR_BUILDER_COMPOSER/745/

**Comment by gundevel** [comment]
Acessibility Sniffer: front page checking WCAG2AA http://openy.cibox.tools/build745/frontWCAG2AAhtmlcs.txt
Acessibility Sniffer: join page checking WCAG2AA http://openy.cibox.tools/build745/joinWCAG2AAhtmlcs.txt
Acessibility Sniffer: locations page checking WCAG2AA http://openy.cibox.tools/build745/locationsWCAG2AAhtmlcs.txt
Acessibility Sniffer: schedules page checking WCAG2AA http://openy.cibox.tools/build745/schedulesWCAG2AAhtmlcs.txt
Acessibility Sniffer: blog page checking WCAG2AA http://openy.cibox.tools/build745/blogWCAG2AAhtmlcs.txt

**Comment by gundevel** [comment]
Build comment file: 
--------------

 
| Environment | Link |
| ------------- | ------------- |
| Fresh OpenY installation - Rose theme | http://openy.cibox.tools/build746 |
| Fresh OpenY installation - Lily theme | http://lily.openy.cibox.tools/build746 |
| Upgraded(upgrade path) installation | http://upgrade.openy.cibox.tools/build746 |
| Installation process | http://install.openy.cibox.tools/build746/install.php |

--------------
Test PASSed.
Refer to this link for build results (access rights to CI server needed): 
http://openy.cibox.tools:8080/job/PR_BUILDER_COMPOSER/746/

**Comment by gundevel** [comment]
Acessibility Sniffer: front page checking WCAG2AA http://openy.cibox.tools/build746/frontWCAG2AAhtmlcs.txt
Acessibility Sniffer: join page checking WCAG2AA http://openy.cibox.tools/build746/joinWCAG2AAhtmlcs.txt
Acessibility Sniffer: locations page checking WCAG2AA http://openy.cibox.tools/build746/locationsWCAG2AAhtmlcs.txt
Acessibility Sniffer: schedules page checking WCAG2AA http://openy.cibox.tools/build746/schedulesWCAG2AAhtmlcs.txt
Acessibility Sniffer: blog page checking WCAG2AA http://openy.cibox.tools/build746/blogWCAG2AAhtmlcs.txt

**Comment by gundevel** [comment]
Build comment file: 
--------------

 
| Environment | Link |
| ------------- | ------------- |
| Fresh OpenY installation - Rose theme | http://openy.cibox.tools/build748 |
| Fresh OpenY installation - Lily theme | http://lily.openy.cibox.tools/build748 |
| Upgraded(upgrade path) installation | http://upgrade.openy.cibox.tools/build748 |
| Installation process | http://install.openy.cibox.tools/build748/install.php |

--------------
Test PASSed.
Refer to this link for build results (access rights to CI server needed): 
http://openy.cibox.tools:8080/job/PR_BUILDER_COMPOSER/748/

**Comment by gundevel** [comment]
Acessibility Sniffer: front page checking WCAG2AA http://openy.cibox.tools/build748/frontWCAG2AAhtmlcs.txt
Acessibility Sniffer: join page checking WCAG2AA http://openy.cibox.tools/build748/joinWCAG2AAhtmlcs.txt
Acessibility Sniffer: locations page checking WCAG2AA http://openy.cibox.tools/build748/locationsWCAG2AAhtmlcs.txt
Acessibility Sniffer: schedules page checking WCAG2AA http://openy.cibox.tools/build748/schedulesWCAG2AAhtmlcs.txt
Acessibility Sniffer: blog page checking WCAG2AA http://openy.cibox.tools/build748/blogWCAG2AAhtmlcs.txt

**Comment by gundevel** [comment]
Build comment file: 
--------------

 
| Environment | Link |
| ------------- | ------------- |
| Fresh OpenY installation - Rose theme | http://openy.cibox.tools/build749 |
| Fresh OpenY installation - Lily theme | http://lily.openy.cibox.tools/build749 |
| Upgraded(upgrade path) installation | http://upgrade.openy.cibox.tools/build749 |
| Installation process | http://install.openy.cibox.tools/build749/install.php |

--------------
Test PASSed.
Refer to this link for build results (access rights to CI server needed): 
http://openy.cibox.tools:8080/job/PR_BUILDER_COMPOSER/749/

**Comment by bjwilkins** [comment]
Build comment file: 
--------------

 
| Environment | Link |
| ------------- | ------------- |
| Fresh OpenY installation - Rose theme | http://openy.cibox.tools/build750 |
| Fresh OpenY installation - Lily theme | http://lily.openy.cibox.tools/build750 |
| Upgraded(upgrade path) installation | http://upgrade.openy.cibox.tools/build750 |
| Installation process | http://install.openy.cibox.tools/build750/install.php |

--------------
Test PASSed.
Refer to this link for build results (access rights to CI server needed): 
http://openy.cibox.tools:8080/job/PR_BUILDER_COMPOSER/750/

**Comment by bjwilkins** [comment]
Build comment file: 
--------------

 
| Environment | Link |
| ------------- | ------------- |
| Fresh OpenY installation - Rose theme | http://openy.cibox.tools/build751 |
| Fresh OpenY installation - Lily theme | http://lily.openy.cibox.tools/build751 |
| Upgraded(upgrade path) installation | http://upgrade.openy.cibox.tools/build751 |
| Installation process | http://install.openy.cibox.tools/build751/install.php |

--------------
Test PASSed.
Refer to this link for build results (access rights to CI server needed): 
http://openy.cibox.tools:8080/job/PR_BUILDER_COMPOSER/751/

**Comment by bjwilkins** [comment]
Build comment file: 
--------------

 
| Environment | Link |
| ------------- | ------------- |
| Fresh OpenY installation - Rose theme | http://openy.cibox.tools/build752 |
| Fresh OpenY installation - Lily theme | http://lily.openy.cibox.tools/build752 |
| Upgraded(upgrade path) installation | http://upgrade.openy.cibox.tools/build752 |
| Installation process | http://install.openy.cibox.tools/build752/install.php |

--------------
Test PASSed.
Refer to this link for build results (access rights to CI server needed): 
http://openy.cibox.tools:8080/job/PR_BUILDER_COMPOSER/752/

**Comment by bjwilkins** [comment]
Acessibility Sniffer: front page checking WCAG2AA http://openy.cibox.tools/build752/frontWCAG2AAhtmlcs.txt
Acessibility Sniffer: join page checking WCAG2AA http://openy.cibox.tools/build752/joinWCAG2AAhtmlcs.txt
Acessibility Sniffer: locations page checking WCAG2AA http://openy.cibox.tools/build752/locationsWCAG2AAhtmlcs.txt
Acessibility Sniffer: schedules page checking WCAG2AA http://openy.cibox.tools/build752/schedulesWCAG2AAhtmlcs.txt
Acessibility Sniffer: blog page checking WCAG2AA http://openy.cibox.tools/build752/blogWCAG2AAhtmlcs.txt

**Comment by podarok** [comment]
retest this please

**Comment by bjwilkins** [comment]
Build comment file: 
--------------

 
| Environment | Link |
| ------------- | ------------- |
| Fresh OpenY installation - Rose theme | http://openy.cibox.tools/build753 |
| Fresh OpenY installation - Lily theme | http://lily.openy.cibox.tools/build753 |
| Upgraded(upgrade path) installation | http://upgrade.openy.cibox.tools/build753 |
| Installation process | http://install.openy.cibox.tools/build753/install.php |

--------------
Test PASSed.
Refer to this link for build results (access rights to CI server needed): 
http://openy.cibox.tools:8080/job/PR_BUILDER_COMPOSER/753/

**Comment by bjwilkins** [comment]
Acessibility Sniffer: front page checking WCAG2AA http://openy.cibox.tools/build753/frontWCAG2AAhtmlcs.txt
Acessibility Sniffer: join page checking WCAG2AA http://openy.cibox.tools/build753/joinWCAG2AAhtmlcs.txt
Acessibility Sniffer: locations page checking WCAG2AA http://openy.cibox.tools/build753/locationsWCAG2AAhtmlcs.txt
Acessibility Sniffer: schedules page checking WCAG2AA http://openy.cibox.tools/build753/schedulesWCAG2AAhtmlcs.txt
Acessibility Sniffer: blog page checking WCAG2AA http://openy.cibox.tools/build753/blogWCAG2AAhtmlcs.txt

**Comment by bjwilkins** [comment]
Build comment file: 
--------------

 
| Environment | Link |
| ------------- | ------------- |
| Fresh OpenY installation - Rose theme | http://openy.cibox.tools/build760 |
| Fresh OpenY installation - Lily theme | http://lily.openy.cibox.tools/build760 |
| Upgraded(upgrade path) installation | http://upgrade.openy.cibox.tools/build760 |
| Installation process | http://install.openy.cibox.tools/build760/install.php |

--------------
Test PASSed.
Refer to this link for build results (access rights to CI server needed): 
http://openy.cibox.tools:8080/job/PR_BUILDER_COMPOSER/760/

**Comment by bjwilkins** [comment]
Acessibility Sniffer: front page checking WCAG2AA http://openy.cibox.tools/build760/frontWCAG2AAhtmlcs.txt
Acessibility Sniffer: join page checking WCAG2AA http://openy.cibox.tools/build760/joinWCAG2AAhtmlcs.txt
Acessibility Sniffer: locations page checking WCAG2AA http://openy.cibox.tools/build760/locationsWCAG2AAhtmlcs.txt
Acessibility Sniffer: schedules page checking WCAG2AA http://openy.cibox.tools/build760/schedulesWCAG2AAhtmlcs.txt
Acessibility Sniffer: blog page checking WCAG2AA http://openy.cibox.tools/build760/blogWCAG2AAhtmlcs.txt

**Comment by gundevel** [comment]
Build comment file: 
--------------

 
| Environment | Link |
| ------------- | ------------- |
| Fresh OpenY installation - Rose theme | http://openy.cibox.tools/build768 |
| Fresh OpenY installation - Lily theme | http://lily.openy.cibox.tools/build768 |
| Upgraded(upgrade path) installation | http://upgrade.openy.cibox.tools/build768 |
| Installation process | http://install.openy.cibox.tools/build768/install.php |

--------------
Test PASSed.
Refer to this link for build results (access rights to CI server needed): 
http://openy.cibox.tools:8080/job/PR_BUILDER_COMPOSER/768/

**Comment by bjwilkins** [comment]
Build comment file: 
--------------

 
| Environment | Link |
| ------------- | ------------- |
| Fresh OpenY installation - Rose theme | http://openy.cibox.tools/build769 |
| Fresh OpenY installation - Lily theme | http://lily.openy.cibox.tools/build769 |
| Upgraded(upgrade path) installation | http://upgrade.openy.cibox.tools/build769 |
| Installation process | http://install.openy.cibox.tools/build769/install.php |

--------------
Test PASSed.
Refer to this link for build results (access rights to CI server needed): 
http://openy.cibox.tools:8080/job/PR_BUILDER_COMPOSER/769/

**Comment by bjwilkins** [comment]
Build comment file: 
--------------

 
| Environment | Link |
| ------------- | ------------- |
| Fresh OpenY installation - Rose theme | http://openy.cibox.tools/build772 |
| Fresh OpenY installation - Lily theme | http://lily.openy.cibox.tools/build772 |
| Upgraded(upgrade path) installation | http://upgrade.openy.cibox.tools/build772 |
| Installation process | http://install.openy.cibox.tools/build772/install.php |

--------------
Test PASSed.
Refer to this link for build results (access rights to CI server needed): 
http://openy.cibox.tools:8080/job/PR_BUILDER_COMPOSER/772/

**Comment by bjwilkins** [comment]
Build comment file: 
--------------

 
| Environment | Link |
| ------------- | ------------- |
| Fresh OpenY installation - Rose theme | http://openy.cibox.tools/build773 |
| Fresh OpenY installation - Lily theme | http://lily.openy.cibox.tools/build773 |
| Upgraded(upgrade path) installation | http://upgrade.openy.cibox.tools/build773 |
| Installation process | http://install.openy.cibox.tools/build773/install.php |

--------------
Test PASSed.
Refer to this link for build results (access rights to CI server needed): 
http://openy.cibox.tools:8080/job/PR_BUILDER_COMPOSER/773/

**Comment by bjwilkins** [comment]
Acessibility Sniffer: front page checking WCAG2AA http://openy.cibox.tools/build773/frontWCAG2AAhtmlcs.txt
Acessibility Sniffer: join page checking WCAG2AA http://openy.cibox.tools/build773/joinWCAG2AAhtmlcs.txt
Acessibility Sniffer: locations page checking WCAG2AA http://openy.cibox.tools/build773/locationsWCAG2AAhtmlcs.txt
Acessibility Sniffer: schedules page checking WCAG2AA http://openy.cibox.tools/build773/schedulesWCAG2AAhtmlcs.txt
Acessibility Sniffer: blog page checking WCAG2AA http://openy.cibox.tools/build773/blogWCAG2AAhtmlcs.txt

**Review by podarok — APPROVED** [review]

**Pull request merged**